src/extensions/spikeamplitudes/SpikeAmplitudesView/SelectUnitsWidget.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Allows user to select units, simply by displaying the table with no metrics rows.